vermouth/processors/apply_rubber_band.py

```diff
@@ -307,12 +285,18 @@ def apply_rubber_band(molecule, selector,
     selection = []
     coordinates = []
     missing = []
+    node_to_idx = {}
+    idx_to_node = {}
+    node_count = 0
     for node_key, attributes in molecule.nodes.items():
```

Comment:
`for node_count, (node_key, attributes) in enumerate(molecule.nodes.items()):`?
And you could replace node_count with node_idx, but no strong preference from my side